builtin/settingtypes.txt

```diff
@@ -452,6 +452,9 @@ fall_bobbing_amount (Fall bobbing) float 0.0
 #    -    pageflip: quadbuffer based 3d.
 3d_mode (3D mode) enum none none,anaglyph,interlaced,topbottom,sidebyside,pageflip
 
+#    Distance where the eyes converge in 3D mode.
+3d_convergence_distance (3D convergence distance) float 1.95
```

Comment:
same unit as the original 3d_parallax_strength. Technically it is in meters. Is 10 Minetest unit == 1 meter?